x/staking/types/keys.go

```diff
@@ -45,6 +46,8 @@ var (
 	UnbondingQueueKey    = []byte{0x41} // prefix for the timestamps in unbonding queue
 	RedelegationQueueKey = []byte{0x42} // prefix for the timestamps in redelegations queue
 	ValidatorQueueKey    = []byte{0x43} // prefix for the timestamps in validator queue
+
+	HistoricalInfoKey = []byte{0x50} // prefix for the historical info
```

Comment:
```suggestion
	HistoricalInfoKeyPrefix = []byte{0x50} // prefix for the historical info
```

Reply:
This does not follow the naming scheme above